lib/common/multithread.c

```diff
@@ -87,11 +87,16 @@ static void init_async(h2o_multithread_queue_t *queue, h2o_loop_t *loop)
 {
     int fds[2];
 
+#ifndef _WIN32
     if (cloexec_pipe(fds) != 0) {
         perror("pipe");
         abort();
     }
     fcntl(fds[1], F_SETFL, O_NONBLOCK);
+#else
+    u_long nonblock = 1;
```

Comment:
> To which part of the callbacks does the client proceed? exmaples/libh2o/http1client.c defines a number of callbacks that are called one by one. on_connect is called first, then on_head, and on_body. You could also set breakpoints to (or emit logs in) the callback functions within socketpool.c or http1client.c to determine which callbacks are called, and which are not.

When using libuv on_connect is called and it doesn't complain and that comes from a call stack of

1 on_connect
2 call_connect_cb
3 uv_process_tcp_connect_req
4 uv_process_reqs
5 uv_run

then after that it hits on_head and exits on line 108 after fprintf "I/O timeout"

the stack for this is

1 on_head
2 on_error_before_head
3 h2o_timeout_run
4 on_timeout

After debugging in fiddler2 the http req debugger I can see that the http request is infact never made to argv[1] in my case http://google.com

now the only strange piece of code left is this
/uv-binding.c.h:147:25: warning: passing argument 1 of 'dup' makes integer from pointer without a cast
     if ((info->fd = dup(fd)) == -1)

I tried using _get_osfhanle but it doesn't change anything :/

> Personally I did not expect somebody to start porting the standalone server to Windows, and am fascinated by how far you have achieved so far.

Someone was bound to try it sooner or later :D